tests/pyreverse/test_inspector.py

```diff
@@ -30,7 +30,14 @@ def project(get_project: GetProjectCallable) -> Generator[Project, None, None]:
     with _test_cwd(TESTS):
         project = get_project("data", "data")
         linker = inspector.Linker(project)
-        linker.visit(project)
+        if "clientmodule_test" in str(project):
```

Comment:
This is never true, as the `project` is always the same here (the `data` directory underneath `/tests`). The string representation of `project` does not contain the substring `clientmodule_test`.

I would suggest just get rid of the `if` condition here.

Reply:
I tried that, but I received many failures. I'll try again tonight. 🤔

Reply:
Hm, but with the current change we are always hitting the `else`, and nothing has changed effectively compared to before. 
I haven't looked at how to actually properly catch the warning, my guess would be that there are some other tests that don't catch this Warning yet (the majority of the legacy tests for `pyreverse` work with the same test data). I will also take a closer look at it this or tomorrow evening!

Reply:
Neither on your branch nor on main could I reproduce the deprecation warnings on the `pyreverse` tests. 
On a [recent CI run](https://github.com/PyCQA/pylint/actions/runs/3514404778/jobs/5888375701) there is only one `DeprecationWarning` for some other test case, but not for `pyreverse`.
Am I missing something here like an option I have to turn on first?

Reply:
Sorry @DudeNr33 you're quite right, the `else` branch always executes. I've just pushed a change to just ignore warnings in this fixture.